server/src/routes/api/v1/squidsRouter.js

```diff
@@ -19,4 +20,14 @@ squidsRouter.get(
   })
 )
 
+squidsRouter.post(
+  "/",
+  nextWrapper(async (req, res) => {
```

Comment:
good! we handle for errors here and return a `201` when successful in inserting the squid